<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>other_viz_dialog</class>
 <widget class="QDialog" name="other_viz_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Additional electrostatics visualizations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Positive Isosurface</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Contour (kT/e):</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>pos_iso_level_spinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QSpinBox" name="pos_iso_level_spinBox"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="pos_iso_checkBox">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Color:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>pos_iso_color_lineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignVCenter">
           <widget class="QLineEdit" name="pos_iso_color_lineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Negative Isosurface</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Contour (kT/e):</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>neg_iso_level_spinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QSpinBox" name="neg_iso_level_spinBox"/>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Color:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>neg_iso_color_lineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignVCenter">
           <widget class="QLineEdit" name="neg_iso_color_lineEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="neg_iso_checkBox">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Field lines</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Ramp:</string>
            </property>
            <property name="buddy">
             <cstring>fieldlines_ramp_lineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignVCenter">
           <widget class="QLineEdit" name="fieldlines_ramp_lineEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="fieldlines_checkBox">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
